src/gen_typescript/templates/Helpers.ts

```diff
@@ -0,0 +1,19 @@
+export type EventListener = (breezEvent: BreezEvent) => void
+
+export type LogStream = (logEntry: LogEntry) => void
+
+export const connect = async (config: Config, seed: number[], listener: EventListener): Promise<EmitterSubscription> => {
+    const subscription = BreezSDKEmitter.addListener("breezSdkEvent", listener)
+    
+    await BreezSDK.connect(config, seed)
+
+    return subscription
+}
```

Comment:
I wonder if there is a better way to do this by partially generating the function and arguments. 

Here the EventListener interface is replaced with a function callback, the return type is replaced with the EmitterSubscription for the emitter and the BreezSDK connect method is called without the listener (is managed on the native side).